opt/scopeserver/dataserver/utils/loom.py

```diff
@@ -397,19 +416,16 @@ def add_user_clustering(self, request) -> Tuple[bool, str]:
 
         new_clusterings = Loom.dfToNamedMatrix(clusterings)
 
-        self.loom_connection = self.lfh.change_loom_mode(
-            self.file_path, mode="r+", partial_md5_hash=self.partial_md5_hash
-        )
+        self.loom_connection = self.lfh.change_loom_mode(self.file_path, mode="r+")
         loom = self.loom_connection
         loom.ca.Clusterings = new_clusterings
         loom.attrs["MetaData"] = json.dumps(metaJson)
-        self.loom_connection = self.lfh.change_loom_mode(
-            self.file_path, mode="r", partial_md5_hash=self.partial_md5_hash
-        )
-        loom = self.loom_connection
+        self.loom_connection = self.lfh.change_loom_mode(self.file_path, mode="r")
+
+        self.update_ss(update="clusterings")
```

Comment:
No, the update now only adds the new entries to the search space rather than rebuilding the entire thing (see https://github.com/aertslab/SCope/pull/360/files#diff-4cb5f7211d42fdedde8dab51db72b61cR79-R86)